nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -152,6 +154,36 @@ NRF_SO_EXCEPTIONAL_DATA
 
    The socket option is supported from modem firmware v2.0.0.
 
+NRF_SO_KEEPOPEN
+   Keep the socket from being closed upon PDN disconnection or reactivation events, or when the device is put into flight mode (``+CFUN=4``).
+   Until the PDN connection is reestablished, the socket is not functional.
+   Output operations such as :c:func:`nrf_send` and :c:func:`nrf_connect` return an error and set ``errno`` to :c:macro:`NRF_ENETUNREACH`.
+   Input operations such as :c:func:`nrf_recv` and :c:func:`nrf_accept` block, since no data can be received, or return no data (if the socket or the operation is non-blocking).
```

Comment:
```suggestion
   Output operations, such as the functions :c:func:`nrf_send` and :c:func:`nrf_connect` return an error and set ``errno`` to :c:macro:`NRF_ENETUNREACH`.
   Input operations, such as the functions :c:func:`nrf_recv` and :c:func:`nrf_accept` block, since no data can be received, or return no data (if the socket or the operation is non-blocking).
```

Reply:
Updated